app/uk/gov/hmrc/singlecustomeraccountwrapperdata/controllers/actions/AuthAction.scala

```diff
@@ -44,16 +44,6 @@ class AuthActionImpl @Inject()(
       if (confLevel.level < ConfidenceLevel.L200.level) Some(confLevel) else None
   }
 
-  private def trimmedRequest[A](request: Request[A]): Request[A] = request
```

Comment:
I don't understand why we had this and why do we remove now ?

Reply:
no idea either, was copy pasted code that wasn't being used, it looks like it cleans requests that are POST's, but there are only GET's in the service now